apps/base/rucio-server/cms-rucio-server.yaml

```diff
@@ -106,9 +106,11 @@ config:
       username: "cmsrucio"
       #config.trace.tracedir: directory where traces are written to (default "/var/log/rucio/trace")
       tracedir: "/var/log/rucio/trace"
+  server:
```

Comment:
You don't need server here, I think. just config.database.pool_size

Reply:
where to config.database.pool_size? 
Another question: there are sessions "tracer_kronos" vs "tracerKronos". How are the configs divided into these two sessions? how the section names are decided?

Reply:
```
config:   
   database:
      pool_size:
```      

Anything in `config:` goes into rucio.cfg and uses the section names there. `tracerKronos` is for the k8s Deployment and includes pod parameters, execution parameters, etc.